### DISCUSSION TITLE
How display of redirected ports on connection (as in previous versions) ?

### DISCUSSION THREAD

**Discussion by 120dev** [Eugeny/tabby#5798] [Q&A] [answered]
Hello, 

In previous versions, when there was a port forwarding, they were displayed on connection, since the latest version there is no more message (green success).

Is there a possibility to have this successful connection log again?

Thank you

**Comment by 120dev** [comment, marked as answer]
thank you for putting back the display of port forwardings when connecting :)

**Comment by lawweiliang** [comment]
@120dev  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
